eng/pipelines/templates/jobs/ci.yml

```diff
@@ -36,7 +36,12 @@ parameters:
 - name: MatrixReplace
   type: object
   default: []
-
+- name: TestTargetDependency
+  type: string
+  default: not-specified
+- name: ProjectListOverrideFile
```

Comment:
I don't think we need this parameter here we can just set it below directly.

Reply:
Done